config/nginx/conf.d/cantusdb.conf

```diff
@@ -46,7 +46,12 @@ server {
     location /media {
         alias /resources/media/;
     }
-    
+
+    location /concordances {
+        alias /resources/api_cache/concordances.json;
+        expires 6h;
```

Comment:
What is the logic behind this choice of expiration time? What about `modified 24h` given that theoretically there should be a new one every 24 hours

Reply:
response headers now look like:
```
HTTP/1.1 200 OK
Server: nginx/1.24.0
Date: Tue, 30 Jan 2024 21:00:10 GMT
Content-Type: application/octet-stream
Content-Length: 192128923
Last-Modified: Tue, 30 Jan 2024 20:46:59 GMT
Connection: keep-alive
ETag: "65b96043-b73a79b"
Expires: Wed, 31 Jan 2024 20:46:59 GMT
Cache-Control: max-age=85609
Accept-Ranges: bytes
```